docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   react:
     build: ./frontend
     container_name: react
-    command: /bin/bash -c "npm update --legacy-peer-deps && npm start"
+    command: npm start
```

Comment:
Just run npm start... and someone will just have to run `npm update` every week so to get updates. But I think this will be easier.